modules.d/00systemd/module-setup.sh

```diff
@@ -94,8 +94,10 @@ install() {
 
     if [[ $hostonly ]]; then
         inst_multiple -H -o \
-            /etc/systemd/system.conf \
-            /etc/systemd/system.conf.d/*.conf \
+            "$systemdutildir"/system.conf \
+            "$systemdutildir"/system.conf.d/*.conf \
```

Comment:
hostonly sections should include only files under /etc

Reply:
Why is that?
Systemd definitely takes host configuration from `/usr`... that's the point of this PR.
Why not to include it, when initrd conf. should ideally behave like the real system in host-only mode?

Reply:
conf shipped by packages => `/usr` => non-hostonly (generic, not specific to any host)
conf created by some admin => `/etc` => hostonly (specific to the host)

I'm not saying that the conf in `/usr` should not be included, but always include it outside of the hostonly check.

Reply:
I agree, the /usr file should be always included